src/pyddx_classes.cpp

```diff
@@ -149,6 +151,7 @@ class Model {
   // Return the holder pointer ... internal function. Use only if you know
   // what you are doing.
   void* holder() const { return m_holder; }
+  void* error() const { return m_error; }
```

Comment:
The "use only if you know what you are doing" applies here as well I'd say.

Reply:
Second comment added.